packages/components/rating/src/Rating.tsx

```diff
@@ -129,7 +129,7 @@ export const Rating = forwardRef<HTMLDivElement, RatingProps>(
 
     return (
       <div
-        className="relative"
+        className="relative inline-flex"
```

Comment:
Why adding this class?

Reply:
As we focus the group, we didn't want to end with a full-width block (thx to the `div`) as our stars only fill a part of it...
Before: ![image](https://github.com/adevinta/spark/assets/66770550/d8cc681c-b2e7-450c-b5f1-183c14a5928e)
After: 
![image](https://github.com/adevinta/spark/assets/66770550/10a1ab45-d162-47ab-9d8d-bc77d3730bb7)

Reply:
I see. Btw, we could achieve the same result by using `w-max` here. I have nothing against using flex here, but since this element only has one true child, we don't take full advantage of the flex layout. Either way works for me 🤷